task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -169,6 +169,10 @@ spec:
       set -o verbose
       set -e
       cd $(workspaces.source.path)
+      if [ -n "$(params.BUILDER_IMAGE)" ]; then
```

Comment:
I think our CI checks are broken - this should have caused a failure :confused: 

https://github.com/redhat-appstudio/build-definitions/blob/bc5eeaa574694a9ea449f38c69039edadc6127a4/.tekton/tasks/yaml-lint.yaml#L35

Reply:
Yeah, I'm getting:

```
Tasks contains params in script section (https://tekton.dev/docs/pipelines/tasks/#substituting-in-script-blocks)
task/acs-deploy-check/0.1/acs-deploy-check.yaml
task/acs-image-check/0.1/acs-image-check.yaml
task/acs-image-scan/0.1/acs-image-scan.yaml
task/buildah/0.1/buildah.yaml
task/buildah-remote/0.1/buildah-remote.yaml
task/s2i-java/0.1/s2i-java.yaml
task/s2i-nodejs/0.1/s2i-nodejs.yaml
task/update-infra-deployments/0.1/update-infra-deployments.yaml
```

With the extracted script from https://github.com/redhat-appstudio/build-definitions/blob/main/.tekton/tasks/yaml-lint.yaml#L41C1-L51C11

Reply:
Filed a bug for this